src/main/java/com/rcv/Tabulator.java

```diff
@@ -395,16 +409,20 @@ private CandidateStatus getCandidateStatus(String candidate) {
     // store result here
     List<String> selectedWinners = new LinkedList<>();
 
-    // If the number of continuing candidates equals the number of seats to fill, everyone wins.
-    if (currentRoundCandidateToTally.size() == config.getNumberOfWinners() - winnerToRound.size()) {
-      selectedWinners.addAll(currentRoundCandidateToTally.keySet());
-    } else { // see if anyone has met or exceeded the threshold
-      // tally indexes over all tallies to find any winners
-      for (BigDecimal tally : currentRoundTallyToCandidates.keySet()) {
-        if (tally.compareTo(winningThreshold) >= 0) {
-          // we have winner(s)
-          List<String> winningCandidates = currentRoundTallyToCandidates.get(tally);
-          selectedWinners.addAll(winningCandidates);
+    // We should only look for more winners if we haven't already filled all the seats.
+    if (winnerToRound.size() < config.getNumberOfWinners()) {
+      // If the number of continuing candidates equals the number of seats to fill, everyone wins.
+      if (currentRoundCandidateToTally.size()
+          == config.getNumberOfWinners() - winnerToRound.size()) {
+        selectedWinners.addAll(currentRoundCandidateToTally.keySet());
+      } else { // see if anyone has met or exceeded the threshold
```

Comment:
I'm not against this, but I'm not sure this formatting is keeping with the style

Reply:
This is actually what the auto-formatter chose. :( Not my favorite.